pages/common/assets/sass/style.scss

```diff
@@ -379,3 +379,13 @@ dl.inline {
     color: $white;
   }
 }
+
+.application-confirm {
```

Comment:
We already have a selector for this (L268) so just bung the additional stuff there.

Reply:
good shout